src/ExpressiveInstaller/config.php

```diff
@@ -39,7 +40,7 @@
     'questions' => [
         'container' => [
             'question'               => 'Which container do you want to use for dependency injection?',
-            'default'                => 3,
+            'default'                => 4,
```

Comment:
Probably, but I'd not considered automation when we first did this, and now we need to. :smile:

We can re-order them for v3 of the skeleton, as that's a time when folks can expect automation might break.